common/src/main/java/org/apache/celeborn/common/meta/FileInfo.java

```diff
@@ -106,8 +110,17 @@ public synchronized long getLastChunkOffset() {
     return chunkOffsets.get(chunkOffsets.size() - 1);
   }
 
-  public synchronized long getFileLength() {
-    return chunkOffsets.get(chunkOffsets.size() - 1);
+  public long getFileLength() {
+    return getBytesFlushed();
+  }
+
+  public long getBytesFlushed() {
+    return bytesFlushed;
+  }
+
+  public long updateBytesFlushed(int numBytes) {
```

Comment:
This function updateBytesFlushed is called by flush which is synchronized by filewriter, so it don't need add synchronized here